packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -108,7 +112,9 @@ export class TileDoctype<T = Record<string, any>> extends Doctype {
      * @param metadata - Genesis metadata
      * @param opts - Additional options
      */
-    static async create<T>(ceramic: CeramicApi, content: T | null | undefined, metadata?: TileMetadataArgs, opts: DocOpts = {}): Promise<TileDoctype<T>> {
+    static async create<T>(ceramic: CeramicApi, content: T | null | undefined, metadata?: TileMetadataArgs, opts: CreateOpts = {}): Promise<TileDoctype<T>> {
+      // sync by default if creating a deterministic document
+      opts = { anchor: true, publish: true, sync: !!metadata?.deterministic, ...opts };
```

Comment:
`opts.sync || !!metadata?.deterministic` maybe ?

Reply:
I don't think that would work if the user explicitly provided `sync: false`?